src/java/test/org/apache/zookeeper/server/quorum/QuorumPeerMainTest.java

```diff
@@ -545,6 +545,7 @@ public void testInconsistentPeerType() throws Exception {
                 defaultedToObserver = true;
             }
             if (warningPresent && defaultedToObserver) {
+                LOG.debug("Content from console is: {}", line);
```

Comment:
"QuorumPeerMainTest" has no reason to fail in `jinkens`, so add a logger to figure out why. I will remove it.